components/core/blocks/Switcher/LanguageSwitcher.vue

```diff
@@ -8,14 +8,13 @@
 </template>
 
 <script>
+import config from 'config';
 import { currentStoreView } from '@vue-storefront/core/lib/multistore';
 const ModalSwitcher = () =>
   import(
     /* webpackChunkName: "vsf-languages-modal" */ 'theme/components/core/blocks/Switcher/Language'
   );
 
-const { i18n = {} } = currentStoreView();
```

Comment:
When this line was executed, `currentStoreView()` returned empty object (without `i18n` key) so `i18n` constant was assigned to `{}`.